src/components/LoginRegister.js

```diff
@@ -0,0 +1,234 @@
+import {
+  VStack,
+  Heading,
+  Button,
+  Box,
+  FormControl,
+  InputGroup,
+  InputLeftElement,
+  InputRightElement,
+  Input,
+  FormHelperText,
+  FormLabel,
+} from '@chakra-ui/react';
+import { useState, useRef } from 'react';
+
+const LoginRegister = (props) => {
+  const [showPassword, setShowPassword] = useState(false);
+  const filePicker = useRef(null);
+  const handleShowPasswordChange = () => {
+    setShowPassword(!showPassword);
+  };
+  return (
+    <VStack w="100%" h="100vh" mx="left" bgGradient="linear(to-r, #E8F6EF, #E8F6EF)">
+      {props.activeOption === 'login' ? (
+        <Box bg="white" mt="10%" textAlign="center" boxShadow="md" p={6} rounded="md">
+          <Heading color="#2FE1D6" mb="8" textShadow="1px 1px #e0e0e0">
+            Welcome
+          </Heading>
+          <form
+            onSubmit={(e) => {
+              e.preventDefault();
+              props.setIsLoggedIn(true);
+            }}
+          >
+            <VStack>
+              <FormControl>
+                <InputGroup>
+                  <InputLeftElement
+                    color="gray.300"
+                    pointerEvents="none"
+                    children={<i className="fas fa-user-alt"></i>}
+                  />
+                  <Input borderColor="#2FE1D6" type="email" placeholder="Email address" />
+                </InputGroup>
+              </FormControl>
+              <FormControl>
+                <InputGroup>
+                  <InputLeftElement
+                    pointerEvents="none"
+                    color="gray.300"
+                    children={<i className="fas fa-lock"></i>}
+                  />
+                  <Input
+                    borderColor="#2FE1D6"
+                    type={showPassword ? 'text' : 'password'}
+                    placeholder="Password"
+                  />
+                  <InputRightElement width="4.5rem">
+                    <Button
+                      _focus="outline-none"
+                      _hover={{
+                        background: '#2FE1D6',
+                      }}
+                      color="white"
+                      textShadow="1px 1px #e0e0e0"
+                      bg="#2FE1D6"
+                      h="1.75rem"
+                      size="sm"
+                      onClick={handleShowPasswordChange}
```

Comment:
Prefer to explicity pass the argument to functions for event handlers like this because it makes it more explicit to know what the params are

```suggestion
                      onClick={(e) => handleShowPasswordChange(e)}
```